src/PrestaShopBundle/Form/Admin/Sell/Supplier/SupplierType.php

```diff
@@ -150,26 +190,28 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('phone', TextType::class, [
-                'empty_data' => '',
+                'label' => $this->trans('Phone', 'Admin.Global'),
+                'help' => $this->trans('Phone number for this supplier.', 'Admin.Catalog.Help'),
                 'required' => false,
                 'constraints' => $this->getPhoneCommonConstraints(),
             ])
             ->add('mobile_phone', TextType::class, [
-                'empty_data' => '',
+                'label' => $this->trans('Mobile phone', 'Admin.Global'),
+                'help' => $this->trans('Mobile phone number for this supplier.', 'Admin.Catalog.Help'),
```

Comment:
Hi! Regarding this wording, I can't see the help text on the software at the moment? Is it really necessary to have a help text there? Wtdy @LouiseBonnard 
![image](https://user-images.githubusercontent.com/70583503/101499814-fafc7680-396d-11eb-9756-06f80d4573e1.png)

Reply:
(I checked 1.7.7 version)

Reply:
I agree, the two phone fields are explicit enough to me.

Reply:
Removed the help from phone and mobile phone.